aiidalab_widgets_base/viewers.py

```diff
@@ -222,7 +229,89 @@ def _download_tab(self):
+        self.render_btn = ipw.Button(description="Render", icon='fa-paint-brush')
+        self.render_btn.on_click(self._render_structure)
+        self.render_box = ipw.VBox(children=[ipw.Label("Render an image with POVRAY:"), self.render_btn])
+
+        return ipw.VBox([self.download_box, self.screenshot_box, self.render_box])
+
+    def _render_structure(self, change=None):
+        """Render the structure with POVRAY."""
+
+        if not isinstance(self.structure, Atoms):
+            return 
+
+        self.render_btn.disabled = True 
+        omat = np.array(self._viewer._camera_orientation).reshape(4, 4).transpose()
+
+        zfactor = norm(omat[0, 0:3])
+        omat[0:3, 0:3] = omat[0:3, 0:3]/zfactor
+
+        bb = deepcopy(self.structure)
+        bb.pbc = (False, False, False)
+
+        for i in bb:
+            ixyz = omat[0:3, 0:3].dot(np.array([i.x, i.y, i.z]) + omat[0:3, 3])
+            i.x, i.y, i.z = -ixyz[0], ixyz[1], ixyz[2]
+        
+        vertices = []
+    
+        cell = bb.get_cell()
+        vertices.append(np.array([0, 0, 0]))
+        vertices.extend(cell)
+        vertices.extend([cell[0]+cell[1], cell[0]+cell[2], cell[1]+cell[2], cell[0]+cell[1]+cell[2]])
+
+        for n, i in enumerate(vertices):
+            ixyz = omat[0:3, 0:3].dot(i + omat[0:3, 3])
+            vertices[n] = np.array([-ixyz[0], ixyz[1], ixyz[2]])
+    
+        bonds = []
+
+        cutOff = neighborlist.natural_cutoffs(bb) # Takes the cutoffs from the ASE database
+        neighborList = neighborlist.NeighborList(cutOff, self_interaction=False, bothways=False)
+        neighborList.update(bb)
+        matrix = neighborList.get_connectivity_matrix()
+
+        for k in matrix.keys():
+            i = bb[k[0]]
+            j = bb[k[1]]
+
+            v1 = np.array([i.x, i.y, i.z])
+            v2 = np.array([j.x, j.y, j.z])
+            midi = v1 + (v2-v1)*Radius[i.symbol]/(Radius[i.symbol] + Radius[j.symbol])
+            bond = Cylinder(v1, midi, 0.2, Pigment('color', np.array(Colors[i.symbol])),
+                            Finish('phong', 0.8,'reflection', 0.05))
+            bonds.append(bond)
+            bond = Cylinder(v2, midi, 0.2, Pigment('color', np.array(Colors[j.symbol])),
+                            Finish('phong', 0.8,'reflection', 0.05))
+            bonds.append(bond)
+
+                
+        edges = []
+        for x, i in enumerate(vertices):
+            for j in vertices[x+1:]:
+                    if norm(np.cross(i-j, vertices[1]-vertices[0])) < 0.001 or norm(np.cross(i-j, vertices[2]-vertices[0])) < 0.001 or norm(np.cross(i-j, vertices[3]-vertices[0])) < 0.001:
+                        edge = Cylinder(i, j, 0.06, Texture(Pigment( 'color', [212/255.0,175/255.0,55/255.0])), 
+                                        Finish('phong', 0.9,'reflection', 0.01))
+                        edges.append(edge)
+
+        camera = Camera('perspective', 'location', [0, 0, -zfactor/1.5], 'look_at', [0.0, 0.0, 0.0])
```

Comment:
```suggestion
        camera = Camera(self._viewer.camera, 'location', [0, 0, -zfactor/1.5], 'look_at', [0.0, 0.0, 0.0])
```

I think this should allow us to make the camera be in accordance with what has been selected in the viewer.

Reply:
> I think this should allow us to make the camera type be in accordance with what has been selected in the viewer.

I have already tried this. But there is some problem with using "orthographic". The camera position seems mess up. I suggested that we use the "perspective" as default now.  I will check the issue and make a seperate PR.

Reply:
ok.

Reply:
Could you please make an issue to not forget about this?